backend/src/main/java/com/darass/darass/auth/oauth/service/OAuthService.java

```diff
@@ -23,13 +23,17 @@
 
     public TokenResponse oauthLogin(String oauthAccessToken) {
         SocialLoginUser socialLoginUser = userInfoProvider.findSocialLoginUser(oauthAccessToken);
-        Optional<SocialLoginUser> foundSocialLoginUser = socialLoginUserRepository.findByOauthId(socialLoginUser.getOauthId());
+        Optional<SocialLoginUser> expectedSocialLoginUser = socialLoginUserRepository.findByOauthId(socialLoginUser.getOauthId());
```

Comment:
```suggestion
        Optional<SocialLoginUser> possibleSocialLoginUser = socialLoginUserRepository.findByOauthId(socialLoginUser.getOauthId());
```

Optional 네이밍을 어떻게 정할지 애매해서 나온 변수명같네요.
이와 관련하여 구글링을 좀 해봤는데, "possible + Type" 형태로 변수명을 지어보는 것은 어떨까요?
https://betterprogramming.pub/useful-tips-for-naming-your-variables-8139cc8d44b5

Reply:
좋은 글 감사합니다. 반영해보겠습니다~